blivet/osinstall.py

```diff
@@ -1094,6 +1099,653 @@ def get(self, key, default=None):
         return self.mappings.get(key, default)
 
 
+class InstallerStorageConfig(Blivet):
```

Comment:
The ``Config`` part of this class' name makes me think it is just some static configuration not an active object like ``Blivet`` is. What about just ``InstallerStorage``? Or ``InstallerBlivet``?